twig_fractal.services.yml

```diff
@@ -2,10 +2,10 @@ services:
   twig.extension.twig_fractal:
     class: Drupal\twig_fractal\TwigFractal
     tags:
-      - { name: twig.extension }
+      - { name: twig.extension , priority: 220 }
   twig.loader.twig_fractal:
-      class: Drupal\twig_fractal\TwigLoader
-      arguments: ['@app.root', '@module_handler', '@theme_handler']
-      tags:
-        # twig.loader.componentlibrary is registered with 200
-        - { name: twig.loader, priority: 210 }
+    class: Drupal\twig_fractal\TwigLoader
+    arguments: [ '@theme_handler','@components.info', '@theme.manager' ]
```

Comment:
This appears to be the only change which would require a new branch/mainline for D9. Is there a way to avoid that and support both versions in a single mainline?

Reply:
I'm not sure if there is a way to handle both 8 and 9 from the mainline. Is there a way for you to support two versions/tags for this module and release the module to Drupal and make the install with composer that can pick the relevant module version based on Drupal version?

Reply:
No, that would require more than one main branch.  But nevermind.  Let‘s just bump the major version with this PR and it should be fine.  There were no formal releases anyway.

Unless my information is outdated, drupal.org does not support repositories hosted on GitHub yet.  And the GitHub package registry does not support Composer yet.  So until either of these two conditions changes you'd need to continue referencing the repository separately in your composer.json.

Reply:
the services yml was updated last week to account he new changes to the components module - https://github.com/netzstrategen/twig_fractal/pull/30#issuecomment-773347515

Reply:
Hello @sun did you have a chance to look at my further comments above and recent commits to the PR? Really looking forward to get this one merged over to the main line!